test/e2e/backup_test.go

```diff
@@ -73,6 +87,11 @@ func testEtcdBackupOperatorForS3Backup(t *testing.T) string {
 	// 4 seconds timeout via retry is enough; duration longer than that may indicate internal issues and
 	// is worthy of investigation.
 	s3Path := ""
+	s3f, err := s3factory.NewClientFromSecret(f.KubeClient, f.Namespace, os.Getenv("TEST_AWS_SECRET"))
```

Comment:
Probably rename `s3f` to `s3Cli` since it's returning a client and not the factory.

Reply:
sounds good.